### PR TITLE
test: Add test for auto create account

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: contracts/test/autoCreateAccount.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
+// const { assert } = require("chai");
+// const { ethers, web3 } = require("hardhat");
+// import { RPC } from "@ckb-lumos/toolkit";
+// import { assert } from "chai"
+// import hardhat, { ethers } from "hardhat"
+// import { ERC20_HEX, ERC20_HEX_ABI } from "../scripts/erc20_hex";
```
Comment (zeroqn) [inline contracts/test/autoCreateAccount.js]: Remove these code?

**Review by zeroqn — APPROVED** [review]

File: contracts/test/autoCreateAccount.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+function u32ToLE(num) {
+  const buf = Buffer.alloc(4);
+  buf.writeUInt32LE(num);
+  return `0x${buf.toString("hex")}`;
+}
+
+async function ethAddressToAccountId(ethAddress, rpc) {
+  if (ethAddress === "0x" || ethAddress === "0x" + "00".repeat(20)) {
+    throw new Error("");
```
Comment (ShookLyngs) [inline contracts/test/autoCreateAccount.js]: Maybe give the error a clear message?

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

File: contracts/test/autoCreateAccount.js (hunk 1 above)
Comment (classicalliu) [reply contracts/test/autoCreateAccount.js]: Ohh, sorry, removed

File: contracts/test/autoCreateAccount.js (hunk 2 above)
Comment (classicalliu) [reply contracts/test/autoCreateAccount.js]: Updated

Comment (ShookLyngs) [reply contracts/test/autoCreateAccount.js]: Did you forget to push the commit?

Comment (classicalliu) [reply contracts/test/autoCreateAccount.js]: > Did you forget to push the commit?

Pushed to wrong branch ... sorry ..

**Review by zeroqn — APPROVED** [review]

**Review by ShookLyngs — APPROVED** [review]

**Pull request merged**